lib/kubernetes/client.rb

```diff
@@ -0,0 +1,51 @@
+require "kubeclient"
+
+module Kubernetes
+  class Client
+    API_VERSION = "v1".freeze
+    API_SERVER = "https://kubernetes.default.svc".freeze
+    SERVICE_ACCOUNT = "/var/run/secrets/kubernetes.io/serviceaccount".freeze
+
+    BEARER_TOKEN_FILE = "#{SERVICE_ACCOUNT}/token".freeze
```

Comment:
Are these actual files on the filesystem? Are they there already?

Reply:
Ah sorry I missed your comment earlier - yeah, they are mounted by Kubernetes into the container when you define that the pod uses a ServiceAccount.